pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +213,16 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("persistentvolumes", "persistentvolumes/status", "storageclasses").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("events", "limitranges", "namespaces", "namespaces/status",
```

Comment:
Quota controls much more than storage. The same person *may* be responsible for both, but isn't necessarily. I'd prefer a separate "quota admin" role over combining unrelated resources in this role